### DISCUSSION TITLE
能否提供一个react版本的支持小屏幕和2k屏幕缩放的screen组件呢

### DISCUSSION THREAD

**Discussion by angleneo** [daidaibg/IofTV-Screen#35] [Ideas]
项目很棒，主要想要自适应缩放的这个效果，能否抽成一个组件，react版本的最好了~

**Comment by daidaibg** [comment]
不可以哦亲亲

---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 发送日期 | 2024年02月02日 16:17 |
| 收件人 | daidaibg/IofTV-Screen ***@***.***> |
| 抄送人 | Subscribed ***@***.***> |
| 主题 | [daidaibg/IofTV-Screen] 能否提供一个react版本的支持小屏幕和2k屏幕缩放的screen组件呢 (Discussion #35) |

项目很棒，主要想要自适应缩放的这个效果，能否抽成一个组件，react版本的最好了~
